src/main/scala/com/fortysevendeg/android/sqlite/statement/SQLDroidPreparedStatement.scala

```diff
@@ -0,0 +1,289 @@
+package com.fortysevendeg.android.sqlite.statement
+
+import java.io.{InputStream, Reader}
+import java.math.BigDecimal
+import java.net.URL
+import java.sql._
+import java.util.Calendar
+
+import com.fortysevendeg.android.sqlite.{SQLDroidConnection, _}
+import com.fortysevendeg.android.sqlite.logging.{AndroidLogWrapper, LogWrapper}
+import com.fortysevendeg.android.sqlite.util.StreamUtils._
+import StatementInfo._
+
+import scala.util.{Failure, Success, Try}
+
+class SQLDroidPreparedStatement(
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/821582810/issues/source?bid=2442814&fileBranchId=2752104#l16)